### ISSUE TITLE
Wrong shop sale reports after logging in

### ISSUE THREAD

**Issue by gkp1** [theseedmc/issues#171]
**Describe the bug**
When you log-in, the shop plugins will show you if a player X has bought Y amount of W resources from you. It will also show the amount of money earned while you were away.

While it always correctly shows the amount of money earned while not in the server, it will always show the wrong or outdated information on WHO and WHAT/HOW MUCH items they bought.

**To Reproduce**
1. I log off
2. "SpecificPlayer" buys 256 red dyes from me, making me 307.20. (real example, yesterday)
3. I log in
4. I see this report
5. I log off for a few hours
6. I log in
7. The server tells me again that "SpecificPlayer" bought 256 red dyes from me and that I've earned ~$1400 dollars while I were away.
8. I see someone bought all my Light Blue Dye and half of my Green Concrete stock. This wasn't reported.

**Expected behavior**
It should tell me that some player bought green concrete and light blue dyes from me instead of Red Dye.

![javaw_2020-04-11_16-24-50](https://user-images.githubusercontent.com/63517877/79053711-aacde880-7c15-11ea-9cd1-c6810ae31085.png)
![javaw_2020-04-11_16-25-08](https://user-images.githubusercontent.com/63517877/79053715-aefa0600-7c15-11ea-93d4-0d870348fee6.png)

**Comment by Chlosman** [comment]
Yeah I have the same thing - every time I log in it keeps saying the same person has bought 455 dirt and that my store is now out of dirt when I have actually filled it again since then. ~mushroom realm
